src/ModelUploader/Program.cs

```diff
@@ -31,25 +32,10 @@
 
 Console.WriteLine("Connecting to Azure Storage...");
 
-var azureStorageConnectionString = GetAzureConnectionString();
-if (string.IsNullOrEmpty(azureStorageConnectionString))
-{
-    PrintHelp(
-        $"ERROR: Couldn't find User Secret named '{UserSecretKey}' in configuration.",
-        $"If you don't have a key configured for the app, follow these steps:",
-        $"1. Go to https://portal.azure.com/",
-        $"2. Go to this subscription: DDFun IaaS Dev Shared Public",
-        $"3. Go to this storage resource: dotnetissuelabelerdata",
-        $"4. Select Access keys",
-        $"5. Copy the Key value for key1 or key2.",
-        $"6. Open a command prompt in this project's directory",
-        $"7. Run: dotnet user-secrets set {UserSecretKey} AZURE_KEY_HERE");
-    return 1;
-}
 BlobContainerClient container;
 try
 {
-    container = new BlobContainerClient(azureStorageConnectionString, blobContainerName: "areamodels");
+    container = new BlobContainerClient(new Uri("https://dotnetissuelabelerdata.blob.core.windows.net/areamodels"), new AzureCliCredential());
```

Comment:
We can use `new DefaultAzureCredential()` here instead. This way it will try to resolve Azure credentials from a number of sources (including CLI) which might work better for people who are already authenticated into Azure in Visual Studio or VS Code.